File: app/src/main/java/com/shindra/LineMapActivity.java
package com.shindra;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentTransaction;

import android.content.Intent;
import android.os.Bundle;

public class LineMapActivity extends AppCompatActivity {

    public LineMapFragment mapFragment;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.linemapactivity);

        Intent intentStopActivity = getIntent();
        String lineName = intentStopActivity.getStringExtra("LINE_NAME");

        mapFragment = new LineMapFragment(lineName);
        /* Initialize transaction */
        FragmentTransaction ft = getSupportFragmentManager().beginTransaction();
        ft.add(R.id.FrameLayoutMap, mapFragment);
        ft.commit();
    }
}

File: app/src/main/java/com/shindra/LineMapActivity.kt
package com.shindra

import Fragments.LineMapFragment
import android.os.Bundle
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity
import androidx.lifecycle.ViewModelProvider

class LineMapActivity: AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.line_map_activity)

        val lineTramName = intent.getStringExtra("lineTramName")

        val lineMapFragment = LineMapFragment()
        val bundle = Bundle()
        bundle.putString("lineTramName", lineTramName)
        lineMapFragment.arguments = bundle
        val fragmentTransaction = supportFragmentManager.beginTransaction()
        fragmentTransaction.add(R.id.lineMapFragment, lineMapFragment).commit()
    }
}